dora/src/cannon/codegen.rs

```diff
@@ -1978,10 +1978,7 @@ where
 
         let arguments = self.argument_stack.drain(..).collect::<Vec<_>>();
         let self_register = arguments[0];
-
-        let bytecode_type_self = self.bytecode.register_type(self_register);
         let position = self.bytecode.offset_position(self.current_offset.to_u32());
-        assert_eq!(bytecode_type_self, BytecodeType::Ptr);
```

Comment:
This still needs to be solved. The first argument needs to be a pointer.

Reply:
Ok, will look into it, thanks!

Reply:
Was some left-over thing, removed it and everything is fine now!